css/master.css

```diff
@@ -334,3 +334,112 @@
   padding-top: 30%;
   text-align: center;
 }
+
+.contact {
+  margin-top: var(--section-main-padding);
+  margin-bottom: var(--section-main-padding);
+}
+
+/* stylelint-disable */
```

Comment:
- [ ] [OPTIONAL] Kindly make sure not to disable linters, its not good practise to disable them unless specified in the project requirements.